src/test/java/org/assertj/core/api/int2darray/Int2DArrayAssert_isDeepEqualTo_Test.java

```diff
@@ -0,0 +1,138 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * Copyright 2012-2020 the original author or authors.
+ */
+package org.assertj.core.api.int2darray;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.BDDAssertions.then;
+import static org.assertj.core.error.ShouldHaveSameSizeAs.shouldHaveSameSizeAs;
+import static org.assertj.core.error.ShouldHaveSameSizeOfSubArrayAs.shouldHaveSameSizeOfSubarrayAs;
+import static org.assertj.core.error.ShouldNotBeNull.shouldNotBeNull;
+import static org.assertj.core.error.array.ShouldBeEqual.shouldBeEqual;
+import static org.assertj.core.presentation.StandardRepresentation.STANDARD_REPRESENTATION;
+import static org.assertj.core.util.AssertionsUtil.expectAssertionError;
+
+import org.assertj.core.api.Int2DArrayAssert;
+import org.assertj.core.description.EmptyTextDescription;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+/**
+ * Tests for <code>{@link Int2DArrayAssert#isDeepEqualTo(int[][])}</code>.
+ *
+ * @author Maciej Wajcht
+ */
+@DisplayName("Int2DArrayAssert isDeepEqualTo")
+class Int2DArrayAssert_isDeepEqualTo_Test {
+
+  @Test
+  void should_pass_if_both_actual_and_expected_are_null() {
+    // GIVEN
+    int[][] actual = null;
+    int[][] expected = null;
+
+    // WHEN, THEN
```

Comment:
minor but better to address it now `WHEN, THEN` -> `WHEN/THEN` and no need of a blank line  before it, thanks!

Reply:
General comment, we usually don't put blank lines before WHEN or THEN unless if it makes the test more clearer. If that's too much work, I'll do it.

Reply:
done